aspnet/mvc/controllers/testing/sample/TestingControllersSample/src/TestingControllersSample/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@using TestingControllersSample.ViewModels
+@model IEnumerable<StormSessionViewModel>
+
+@{
+    ViewBag.Title = "Brainstormer";
+    Layout = "_Layout";
+}
+
+<h2>BrainStorm Sessions</h2>
+<div class="row">
+    <div class="col-md-9">
+        @foreach (var storm in Model)
+        {
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <a href="@Url.Action("Index","Session", new {id=@storm.Id})">@storm.Name</a> - @storm.DateCreated.ToString()
+                </div>
+                <div class="panel-body">
+                    @storm.IdeaCount ideas
+                </div>
+            </div>
+        }
+    </div>
+    <div class="col-md-3">
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                Add New Session
+            </div>
+            <div class="panel-body">
+                @Html.ValidationSummary()
+                <form method="post">
```

Comment:
I think you'd better use the Form Tag Helper here so it generates an XSS token. I don't think @blowdart  will allow us to publish code without the token as customer will copy your code to production.